modules/making-open-source-more-inclusive.adoc

```diff
@@ -0,0 +1,6 @@
+[preface]
+
+[id="making-open-source-more-inclusive"]
+= Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
```

Comment:
Just a nitpick but IMO it's best practice to put each sentence in a paragraph on a separate line. If we're going to make this our own module, I'd propose we make that change here.